extensions/ql-vscode/src/vscode-tests/run-integration-tests.ts

```diff
@@ -112,18 +112,21 @@ function getLaunchArgs(dir: TestDir) {
   switch (dir) {
     case TestDir.NoWorksspace:
       return [
-        '--disable-extensions'
+        '--disable-extensions',
```

Comment:
These `--disable-gpu` lines aren't mentioned in the commit message or PR are these meant to be part of this PR?

Reply:
Ah...Let me update the commit message. `--disable-gpu` also removes some error messages from the logs.